Adafruit_FONA.cpp

```diff
@@ -86,6 +86,9 @@ boolean Adafruit_FONA::begin(Stream &port) {
   // turn on hangupitude
   sendCheckReply(F("AT+CVHU=0"), ok_reply);
 
+  // turn on SSL support
+  sendCheckReply(F("AT+CVHU=0"), ok_reply);
```

Comment:
@ladyada  `AT+CVHU` is the voice hangup control according to the command set. Where is it specified that this turns on SSL support?

Reply:
oh blah that shouldnt be here

Reply:
ok re-committed, may want to turn off whitespace comparison.